src/components/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+// import { render } from "react-dom";
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+// App should implement the render function.
+// The render function should display Display and ButtonPanel.
+// REMEMBER! JSX will not compile unless your render function
+// returns a single element. Wrap the children components with
+// div tag and give it an appropriate ID
+
+function App() {
+  // render()
```

Comment:
Remove this if it's not needed. Don't leave unused code as comments in your codebase to keep it clean 🙂

Reply:
Thank you, I removed all unnecessary comments.